python/test/unit/instrumentation/test_vector-add.py

```diff
@@ -0,0 +1,46 @@
+import torch
+
+import pytest
+import os
+
+import triton
+import triton.language as tl
+
+test_stdout = 'Hello From First Instruction of AMDGPU Kernel: add_kernel\ttest_vector-add.py:19:24\n'
+
+@triton.jit
+def add_kernel(x_ptr,  # *Pointer* to first input vector.
```

Comment:
Does this even need to be an add kernel? A simple pass through would do it? We are just testing the "infra" anyway so it should kick in no matter what?

Reply:
Removed add kernel and just made it a simple walk through 3 "empty" kernels.